java/java.iml

```diff
@@ -3,7 +3,7 @@
   <component name="ModuleRunConfigurationManager">
     <shared />
   </component>
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_11">
```

Comment:
We still target Java 11

Reply:
Since we use the Bazel IntelliJ plugin, I think we can actually delete this file entirely. I'll do some checks.

Reply:
@shs96c Thanks to review, and i'd revert that line to JDK_11.

But selenium's ci runs via Java 17.
So It in order not to get confused, we can considor to manage language level on workspace root. (not only java. maybe like nvmrc in node.js projects?)